doc/tech/release_notes/baldy.md

```diff
@@ -0,0 +1,121 @@
+
+<a name="rm"></a>
+## Reference Model (v3.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.rm.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.rm.2` 	| Limiting infrastructure profiles to Basic and Network Intensive (Parking Compute intensive) | Ch02, Ch04, and Ch05 |
+| `baldy.rm.3` 	| Finalising Compliance, Verification and Conformance Strategy | Ch08 	|
+| `baldy.rm.4*` 	| Full Container support 	| Ch04 	|
+| `baldy.rm.5` 	| Complete Security Chapter | Ch07: 100% alignment with ONAP 	|
+| `baldy.rm.6` 	| Virtual Networking/ Networking Fabric Modelling 	| Ch03 	|
+| `baldy.rm.7` 	| Generic Installer Model 	| Ch09 	|
+| `baldy.rm.8` 	| Guidelines 	| Appnedix-A 	|
+
+*Baldy Release should include at least features up to and including `baldy.rm.4`.
+
+<a name="ra1"></a>
+## Reference Architecture 1 (v2.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.ra1.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.ra1.2` 	| Clarify OpenStack version policy 	| Ch01, Ch05 	|
+| `baldy.ra1.3` 	| Align Requirement to RM requirements 	| Ch02 	|
+| `baldy.ra1.4` 	| Create a proposal for an exiting Gap 	|  Ch08	|
+| `baldy.ra1.5` 	| Complete High Level Architecture 	| Ch03 	|
+| `baldy.ra1.6*` 	| Complete Interfaces & APIs 	|  Ch05	|
+| `baldy.ra1.7` 	| Complete Components Level Architecture 	| Ch04 	|
+| `baldy.ra1.8` 	| Complete Security Chapter 	|  Ch06	|
+| `baldy.ra1.9` 	| Complete LCM Chapter 	|  Ch07	|
+
+*Baldy Release should include at least features up to and including `baldy.ra1.6`.
+
+
+<a name="ri1"></a>
+## Reference Implementation 1 (v1.0)
+
+<p align="center"><img src="../figures/ri1_baldy.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 1:</b> RI-1 Baldy Release plan</p>
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.ri1.1` 	| General Cleanups 	| All Chapters 	|
+| `baldy.ri1.2` 	| Complete Overall Requirements 	| Ch02 	|
+| `baldy.ri1.3` 	| Complete Lab Requirements 	| Ch04 	|
+| `baldy.ri1.4` 	| Complete Target State & Metadata 	| Ch03 	|
+| `baldy.ri1.5` 	| Complete Installer Requirements 	| Ch05 	|
+| `baldy.ri1.6` 	| Complete Lab Cookbook (Ops) 	| Ch06 	|
+| `baldy.ri1.6` 	| Restructure & Complete Integration Cookbook	| Ch07 	|
+| `baldy.ri1.7*` 	| Implement Profiles within OPNFV Installers and consume CNTT metadata 	|  **DEV**	|
+
+*Baldy Release should include at least features up to and including `baldy.ri1.7`.
+
+<a name="rc1"></a>
+## Reference Conformance 1 (v1.0-Pre-alpha)
+
+<p align="center"><img src="../figures/rc1_baldy.png" alt="scope" title="Document Types" width="100%"/></p>
+<p align="center"><b>Figure 2:</b> RC-1 Baldy Release plan</p>
```

Comment:
Is there a need to successfully pass RC-1 across three vendor implementations of RI-1?

Reply:
@mf4716 I would say no!  That kind of what the trials will validate.

Reply:
Do we have a Trials lead, or WS, to manage the effort?  We were trying to manage this delivery item from the RI/RC WSs, at least on an interim basis.  RI/RC will need to maintain close alignment with the Trial WS.

Reply:
We shouldn't forget that the tests selected by CNTT has been run for years.
We can expect the vendor implementations to pass RC1. Then we will check case by case if it's about CNTT missing requirements, OpenStack integration bugs or testing failures.

Reply:
I would add that CNTT RI Baldy must pass RC too (ri1.8?)

Reply:
@collivier added.
@mf4716 I would say Bob WS is the owner. let us discuss this in the Adoption WS.